pkgs/applications/misc/xneur/default.nix

```
@@ -21,6 +21,14 @@ stdenv.mkDerivation {
     gst_all_1.gst-plugins-base gst_all_1.gst-plugins-good
   ];
 
+  patches = [
+    (fetchpatch {
+      name = "gcc-10.patch";
+      url = "https://salsa.debian.org/debian/xneur/-/raw/da38ad9c8e1bf4e349f5ed4ad909f810fdea44c9/debian/patches/gcc-10.patch";
+      sha256 = "0sgwb6iab5i9krar22g90rm294c3a99hlnszyr5gca9fbqqcrb95";
```

```suggestion
      sha256 = "0pc17a4sdrnrc4z7gz28889b9ywqsm5mzm6m41h67j2f5zh9k3fy";
```
fetchpatch will do additional normalization

---

Or, as this is a static file, you can also just use `fetchurl`.

---

Done